src/thor/astar.cc

```diff
@@ -160,15 +161,15 @@ void AStarPathAlgorithm::ExpandForward(GraphReader& graphreader,
     }
 
     // Compute the cost to the end of this edge
-    Cost newcost = pred.cost() + costing_->EdgeCost(directededge, tile) +
-                   costing_->TransitionCost(directededge, nodeinfo, pred);
+    auto edge_cost = costing_->EdgeCost(directededge, tile);
```

Comment:
in my  other now defunct PR, this was the part that i screwed up, not keeping just the edge cost separate so you could scale it below. good catch!